eng/scripts/Language-Settings.ps1

```diff
@@ -122,6 +122,23 @@ function Get-javascript-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   return $resultObj
 }
 
+function Get-javascript-DocsMsMetadataForPackage($PackageInfo) { 
+  New-Object PSObject -Property @{ 
+    DocsMsReadMeName = $PackageInfo.Name -replace "^@azure/" , ""
```

Comment:
@praveenkuttappan here is another cases where we might need to do something better for the name replacement.